### PR TITLE
Update to latest 1.16.5-R0.1-SNAPSHOT dependency.

### PR DESCRIPTION
**Pull request by benwoo1110**
This will break support for 1.14 and below.

* Add minidev json dependency.
* Remove use of travel agent.

I need this for some 5.0.0 PRs.

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseCore/MVWorld.java [hunk 1]
```diff
@@ -20,28 +20,20 @@
...
 import me.main__.util.SerializationConfig.NoSuchPropertyException;
 import me.main__.util.SerializationConfig.VirtualProperty;
-import org.bukkit.ChatColor;
-import org.bukkit.Difficulty;
-import org.bukkit.GameMode;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.World;
+import net.minidev.json.JSONObject;
+import org.bukkit.*;
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseCore/MVWorld.java:24]: Wildcard import here.

File: pom.xml [hunk 2]
```diff
@@ -311,6 +307,12 @@
             <artifactId>CommandHandler</artifactId>
             <version>11</version>
         </dependency>
+        <!-- End of CommandHandler Dependency -->
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
+            <version>2.3</version>
```
Comment (nicegamer7) [inline pom.xml:314]: I'm not sure if version 2.3 is compatible. See here, https://netplex.github.io/json-smart/, it says version 2.X is a new interface.

Comment (benwoo1110) [reply pom.xml:314]: I dont facing any issues with `mv version` or paste services. Any known concern about using v2?

**Comment by Brokkonaut** [comment]
You should add respawning at the respawn anchor when updating to 1.16: https://github.com/Brokkonaut/Multiverse-Core/commit/126e005057c33e39da784ea2bcec863a71d620ac#diff-007320f3c3d7e7ef72ba8edfcf728c30ceb1e940504eff3378ea96f43cebe0e7R73

**Comment by benwoo1110** [comment]
> You should add respawning at the respawn anchor when updating to 1.16: [Brokkonaut@126e005#diff-007320f3c3d7e7ef72ba8edfcf728c30ceb1e940504eff3378ea96f43cebe0e7R73](https://github.com/Brokkonaut/Multiverse-Core/commit/126e005057c33e39da784ea2bcec863a71d620ac#diff-007320f3c3d7e7ef72ba8edfcf728c30ceb1e940504eff3378ea96f43cebe0e7R73)

will be a separate PR

File: pom.xml (hunk 2 above)
Comment (nicegamer7) [reply pom.xml:314]: If it works the same then I guess there's no issue.

**Pull request closed without merging**